.github/workflows/autotag.yaml

```diff
@@ -0,0 +1,15 @@
+name: Bump version
```

Comment:
How does it behave with release candidates? Our current version is [v2.0.0-rc.112](https://github.com/wundergraph/graphql-go-tools/releases/tag/v2.0.0-rc.112) In my opinion this isn't a blocker yet, we can also iterate with patch, minor and majors. @devsergiy